### ISSUE TITLE
Multiline secret values

### ISSUE THREAD

**Issue by tevert** [bwhaley/ssmsh#19]
I don't see this currently addressed in the examples - multiline values are output with `\n`, but attempting to `put` with `\n` seemed to result in a literal backslash + n in the value. The AWS CLI seems to require using a file as the payload value in order to represent newlines, which makes me guess this wouldn't be a super-easy feature to add

**Comment by bwhaley** [comment]
Oops, didn't mean to close this.

This is actually supported with the following syntax (which I've just added to the docs):

```
/>put name=/secrets/key/private type=SecureString value="-----BEGIN RSA PRIVATE KEY-----\
... data\
... -----END RSA PRIVATE KEY-----"
Put /secrets/key/private version 1
```

However, in looking in to this, I noticed an issue where an extraneous space was added to each line. There's a fix in [v1.4.2](https://github.com/bwhaley/ssmsh/releases/tag/v1.4.2).

Does this address your question?

**Comment by tevert** [comment]
Yes, that's perfect, thanks!
